tests/integrations/launchdarkly/test_launchdarkly.py

```diff
@@ -202,3 +204,43 @@ def test_launchdarkly_integration_did_not_enable(monkeypatch):
     monkeypatch.setattr(client, "is_initialized", lambda: False)
     with pytest.raises(DidNotEnable):
         LaunchDarklyIntegration(ld_client=client)
+
+
+@pytest.mark.parametrize(
+    "use_global_client",
+    (False, True),
+)
+def test_launchdarkly_span_integration(
+    sentry_init, use_global_client, capture_events, uninstall_integration
+):
+    td = TestData.data_source()
+    td.update(td.flag("hello").variation_for_all(True))
+    td.update(td.flag("world").variation_for_all(True))
+    # Disable background requests as we aren't using a server.
+    config = Config(
+        "sdk-key", update_processor_class=td, diagnostic_opt_out=True, send_events=False
+    )
+
+    uninstall_integration(LaunchDarklyIntegration.identifier)
+    if use_global_client:
+        ldclient.set_config(config)
+        sentry_init(traces_sample_rate=1, integrations=[LaunchDarklyIntegration()])
```

Comment:
`traces_sample_rate` is typed as `Optional[float]`

https://github.com/getsentry/sentry-python/blob/ef0dc37eb666bbd7e2d68f67d1ca62d10c640b4f/sentry_sdk/consts.py#L528

So please pass `1.0` instead of just `1`

```suggestion
        sentry_init(traces_sample_rate=1.0, integrations=[LaunchDarklyIntegration()])
```